pkg/ibmvpc/parser.go

```diff
@@ -14,9 +14,96 @@ import (
 	vpcmodel "github.com/np-guard/vpc-network-config-analyzer/pkg/vpcmodel"
 )
 
+//nolint:funlen,gocyclo //styles are too long and can not be split
+func FilterByResourceGroup(rc *datamodel.ResourcesContainerModel, resourceGroup string) {
```

Comment:
```suggestion
func filterByResourceGroup(rc *datamodel.ResourcesContainerModel, resourceGroup string) {
```
Should this function be exported?